example/main.go

```diff
@@ -12,41 +11,40 @@ import (
 
 func main() {
 	vecty.RenderBody(&body{
-		store: vstore.CreateStore(&store.State{}),
+		store: vstore.New(&store.State{}),
 	})
 }
 
 type body struct {
 	vecty.Core
-	store *vstore.Store
+	store vstore.Store
 }
 
 func (b *body) Render() *vecty.HTML {
 	return elem.Body(
-		b.store.NewComponent(NewMainComp),
+		b.store.Connect(NewMainComp()),
 	)
 }
 
 type mainComp struct {
 	vecty.Core
-	Number   int `vecty:"prop"`
-	Dispatch func(action interface{})
+	store vstore.Store
 }
 
 // NewMainComp is the mainComp constructor
-func NewMainComp(s *vstore.Store) vecty.Component {
-	state := s.GetState().(*store.State)
+func NewMainComp() vstore.StoreComponent {
+	return &mainComp{}
+}
 
-	return &mainComp{
-		Number:   state.Number,
-		Dispatch: s.Dispatch,
-	}
+func (comp *mainComp) Connect(store vstore.Store) {
```

Comment:
should be `(m *mainComp)` like the rest of the methods below. Or the other way around :)

Reply:
Yes, I'll keep it consistent.